lms/templates/courseware/gradebook.html

```diff
@@ -74,13 +74,24 @@
         <thead>
           <tr> <!-- Header Row -->
             %for section in templateSummary['section_breakdown']:
-              <th><div class="assignment-label">${section['label']}</div></th>
+              <%
+                tooltip_str = section['detail']
+                # We are making header labels from the first student record. So for tool tip (title),
+                # I am processing this string ```section['detail']``` from student record and removing
+                # all student related data i.e marks, percentage etc to get only the title of homework.
```

Comment:
Thanks for the comment. I'm wondreing if you can get this in a better way. What if we change the format of the student record, then this breaks.